libbeat/scripts/generate_index_pattern.py

```diff
@@ -71,6 +71,18 @@ def field_to_json(fields, desc, path, output,
     else:
         field["type"] = "string"
 
+    if "enable" in desc and not desc["enabled"]:
+        # Skip objects which are not enabled
+        return
+
+    if "index" in desc and not desc["index"]:
+        # If fields is not indexed, mark it in Kibana
+        field["indexed"] = False
+        field["analyzed"] = False
+        field["doc_values"] = False
+        field["searchable"] = False
+        field["aggregatable"] = False
```

Comment:
great you are fixing this, I already invested some time today in checking why the it still shows up in the index pattern.

You mentioned manual testing, could you also check that those fields are not shown any more in the auto-completion feature when writing a query in kibana. It is weird to get the suggestion, but then nothing is found when using it in the query.